dali/base/dadfs.hpp

```diff
@@ -494,6 +497,10 @@ class DistributedFilePropertyLock {
     {
         return file->queryAttributes();
     }
+    IPropertyTree &queryHistory()
+    {
+        return file->queryHistory();
+    }
```

Comment:
queryHistory() does not feel right as a member of DistributedFilePropertyLock
It's questionable whether queryAttributes() should be either.
There's also a danger of gradually adding more and more methods here.

DistributedFilePropertyLock locks the file so that it can be altered, not just the attributes.

It would be better to add a : 
```
IDistributeFile &queryFile()
```

and use lock.queryFile().queryHistory()